account_invoice_validation_queued/wizards/account_invoice_confirm_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="account_invoice_confirm_view" model="ir.ui.view">
+        <field name="model">account.invoice.confirm</field>
+        <field name="inherit_id" ref="account.account_invoice_confirm_view"/>
```

Comment:
Don't you need a group in this view?

Reply:
Group? This is only adding a button...

Reply:
Yes, but don't you need to have queue permissions to enqueue invoices?

Reply:
The enqueue operation is done with `sudo`, as the permission is not needed for that. If you are an accountant, you want to have invoices validated on parallel without needing to have job permissions.